aws/cloudfront/default-index-redirect.tf

```diff
@@ -13,6 +13,11 @@ data "aws_iam_policy_document" "redirector_assume_role" {
   }
 }
 
+provider "aws" {
+  alias  = "lambda_edge_region"
+  region = "us-east-1"
+    }
```

Comment:
Can you quickly fix the formatting here.

Reply:
Oops, done.